code/game/atoms.dm

```diff
@@ -118,6 +118,9 @@
 	///The color this atom will be if we choose to draw it on the minimap
 	var/minimap_color = MINIMAP_SOLID
 
+	///The current acid on this atom
+	var/obj/effect/xenomorph/acid/atom_current_acid
```

Comment:
Ill let this fly if you delete the old versions of this var

Reply:
@Zar000 I said just use a signal or something instead of an atom var.